### DISCUSSION TITLE
Resize Kit window with tools as Window Magnet

### DISCUSSION THREAD

**Discussion by abtris** [johnlindquist/kit#858] [Ideas]
Hi,
 I'm using keyboard shortcuts for placing and resizing windows on macOS. There are many tools for that as [Magnet](https://magnet.crowdcafe.com/), but with Kit I can't do it. Looks like the tool doesn't see the kit's window as other apps in macOS. 

This helps accessibility by the keyboard. I enlarge fonts that work great! and easy move its position by that don't be cover by others windows will be great. I see the possibility move window by Kit, but it will be great to have one tool for moving windows across macOS.

Is it an issue in the tool, or I'm doing something wrong? Is its window not accessible by MacOS?

**Comment by johnlindquist** [comment]
The Kit window is a "Panel" like spotlight/alfred/raycast/etc. So osx doesn't see it as an active Window.

> A "Panel" is a special type of Window that doesn't "steal focus" from your frontmost app. You'll notice when you open Kit, your previous app still has a blinking cursor.

What's your use-case for leaving Kit open long enough that you want to organize is as a normal window?

I could try to add a setting to "Toggle to window", but that would disable features like "getSelectedText()", "setSelectedText()", and others which require a Panel.
